### ISSUE TITLE
[Feature Request] Allow custom classes for each button

### ISSUE THREAD

**Issue by ellmck** [kamranahmedse/driver.js#202]
It would be nice if it was possible to add custom classes to each button so that they can be styled consistently with the web page.

For example:

```
// current version
var driver = new Driver({ 
     animate: true,
     nextBtnText: "Next",
     prevBtnText: "Previous",
)};
```

```
//new feature of adding classes to buttons
var driver = new Driver({ 
     animate: true,
     nextBtnText: "Next",
     nextBtnClass: "btn btn-primary",
     prevBtnText: "Previous",
     prevBtnClass: ""btn btn-secondary"
)};
```

**Comment by jankapunkt** [comment]
Releated PR: https://github.com/kamranahmedse/driver.js/pull/274

**Comment by kamranahmedse** [comment]
We just released [v1.0](https://github.com/kamranahmedse/driver.js/releases/tag/1.0.3) of driver.js which has been rewritten from the ground up. Please have a [look at the theming docs](https://driverjs.com/docs/theming/) and do let me know if you face any issues.
